remo/featuredrep/models.py

```diff
@@ -27,9 +27,10 @@ def save(self, *args, **kwargs):
 
         # This allows to override the updated_on through the admin interface
         self.updated_on = kwargs.pop('updated_on', None)
-
+        
         if not self.updated_on:
             self.updated_on = now()
         if not self.pk:
             self.created_on = now()
-        super(FeaturedRep, self).save(*args, **kwargs)
+        if self.created_on < now() + datetime.timedelta(days=30):
```

Comment:
This will just block the update without notifying the users why this happened. What we need to do is to validate this field in the clean method in the form and inform users that they cannot use dates more than a month in the future than the current date.

Reply:
You are right, there is a way to report issues on the backend?
In the clean method is the right way?

Reply:
Correct, you can add either a `def clean` method in the form [0] or you can clean only this specific field by adding a `def clean_created_on` method in the same form.

[0] https://github.com/mozilla/remo/blob/master/remo/featuredrep/forms.py#L9